src/constants.js

```diff
@@ -100,6 +100,12 @@ export const MIN_JOB_ESTIMATION = 1;
  */
 export const PIN_SIZE = 6;
 
+/**
+ * Default Hathor Header height, used for recalculating layout offsets.
+ * @type {number}
+ */
+export const HEADER_HEIGHT = 56;
```

Comment:
This should be removed in the iOS color PR

Reply:
The `utils.js` still uses it on the 326 color PR. It will be completely removed soon on the colors refactor:
- #327